client/src/components/GlobalStyledComponents/ReversimStyledComps.jsx

```diff
@@ -30,36 +33,106 @@ export const Container = styled.div`
 `
 
 export const BreakLine = styled.hr`
-  ${ ({ theme: { color, space } }) =>`
-      width: 100%;
+  ${ ({ theme: { color, space, mq } }) =>`
       flex-grow: 1;
       align-self: center;
       margin-left: ${space.m};
-      border-top: 1.5px solid ${color.background_2};
+      border-top: 1.5px solid ${color.box_shadow_1};
+
+      @media (max-width: ${mq.m}){
+        display: none;
+      }
     `}
 `;
 
+export const FAIcon = styled(FontAwesomeIcon)`
+  ${ ({ theme: { color, space } }) => `
+    margin: 0 ${space.m};
+    color: ${color.font_awsome_trash};
+    cursor: pointer;
+  `}
+`;
+
+//Backgrounds only
+
+export const Bkgr4 = styled.div`
+  ${ ({theme: { color } }) => `
+  background-color: ${color.background_4};
+  `}
+`;
+
+//<h2, 3, ...> and <p>
+
+export const HeadingAligner = styled.div`
+  ${({ theme: { space, mq } }) => `
+  display: flex;
+  align-items: center;
+  margin-bottom: calc(2 * ${space.l});
+
+  @media (max-width: ${mq.m}){
+      text-align: center;
+    }
+  `}
+`;
+
+export const HeadingDiamond = styled.img`
+  ${ ({ theme: { space, mq } }) => `
+    width: 85px;
+    height: 85px;
+    margin-right: -${space.xxl};
+    
+    @media (max-width: ${mq.m}){
+      display: none;
+    }
+  `}
+`;
+
+export const HeadingCircle = styled.svg`
```

Comment:
are those `HeadingCircle` and `HeadingDiamond` going to be used in other components beside `PremiumSponsors` and `CommunitySponsors` ?

Im asking because if they belong only there maybe its worth having them only there